### DISCUSSION TITLE
Using Depthmap-script with deforum for stereoscopic videos?

### DISCUSSION THREAD

**Discussion by hoagiemc** [thygate/stable-diffusion-webui-depthmap-script#448] [Q&A]
Hi everybody,

Does somebody know a way how to use this depthmap script with a1111 and deforum to create stereoscopic videos?
I cannot figure it out, any help would be appreciated.

Tnx.

Regards,
Hoagie

**Comment by sonakravtsenko3** [comment]
Hi! I guess you can first create a regular video and then make a stereo video from it. For the second step use video mode.
